dartagnan/src/test/resources/VULKAN-DR-expected.csv

```diff
@@ -68,14 +68,3 @@ litmus/VULKAN/DR/test6-filter.litmus,1
 litmus/VULKAN/DR/test7-filter.litmus,0
 litmus/VULKAN/DR/test9-filter.litmus,1
 litmus/VULKAN/DR/waw-filter.litmus,1
-litmus/VULKAN/DR/Manual/ticketlock-same-wg.litmus,1
-litmus/VULKAN/DR/Manual/ticketlock-diff-wg.litmus,0
-litmus/VULKAN/DR/Manual/ticketlock-acq2rlx-1.litmus,1
-litmus/VULKAN/DR/Manual/ticketlock-acq2rlx-2.litmus,0
-litmus/VULKAN/DR/Manual/ticketlock-rel2rlx.litmus,0
-litmus/VULKAN/DR/Manual/MP-avvis.litmus,1
-litmus/VULKAN/DR/Manual/MP-no-avvis.litmus,0
-litmus/VULKAN/DR/Manual/MP-mesa.litmus,1
-litmus/VULKAN/DR/Manual/MP-mesa-load-acq.litmus,1
-litmus/VULKAN/DR/Manual/MP-mesa-fence-loop.litmus,1
-litmus/VULKAN/DR/Manual/MP-mesa-optimized.litmus,1
```

Comment:
We still want to test those benchmarks. It is ok you removed the files, but in the expected file you need to use the `VULKAN/Manual` tests